test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -45,6 +45,11 @@ SMTCheckerTest::SMTCheckerTest(string const& _filename): SyntaxTest(_filename, E
 
 	m_modelCheckerSettings.solvers &= ModelChecker::availableSolvers();
 
+	/// Underflow and Overflow are not enabled by default for Solidity >=0.8.0,
+	/// so we explicitly enable them for the tests.
+	m_modelCheckerSettings.targets.setFromString("underflow");
+	m_modelCheckerSettings.targets.setFromString("overflow");
```

Comment:
A function that enables all targets might be handy here.

Reply:
That's the old `all` that now won't enable these 2 anymore

Reply:
But I mean adding a new function that specifically always enables all of them. This way you would not have to remember to add new targets here if more of them ever become non-default.

Reply:
Added `all` that ignores the default and just adds everything